torch/jit/_script.py

```diff
@@ -841,7 +849,7 @@ def call_prepare_scriptable_func(obj):
     memo: Dict[int, torch.nn.Module] = {}
     return call_prepare_scriptable_func_impl(obj, memo)
 
-def script(obj, optimize=None, _frames_up=0, _rcb=None):
+def script(obj, optimize=None, _frames_up=0, _rcb=None, example_inputs: Optional[List[Tuple]]=None):
```

Comment:
As discussed earlier, let's avoid publishing it to `torch.jit.script` immediately. We should test against `torch.jit._script` during development. In the future, we can merge `_script` with `script`.

And, I suggest we rename `_script` to something like `_script_pdt` so that no one accidentally uses it.

Reply:
Fixed now